.ci/magician/cloudbuild/build_trigger.go

```diff
@@ -1,3 +1,18 @@
+/*
+* Copyright 2023 Google LLC. All Rights Reserved.
```

Comment:
Generally it's nice to split changes like this into separate PRs to make review easier.